source/_docs/installation/vagrant.markdown

```diff
@@ -44,46 +44,57 @@ $ git config --global core.autocrlf input
 ## {% linkable_title Create the Vagrant VM and start Home Assistant %}
 
 ```bash
-$ vagrant up
+$ ./provision.sh setup
 ```
 
-This will download and start a virtual machine using Virtualbox, which will internally setup the development environment necessary to start Home Assistant process and run test suite as well. After the VM has started successfully, the Home Assistant frontend will be accessible locally from your browser at [http://localhost:8123](http://localhost:8123)
+This will download and start a virtual machine using Virtualbox, which will internally setup the development environment necessary to start Home Assistant. The whole process might take up to 30' to complete, depending on Internet connection speed and workstation resources. After the VM has started successfully, the Home Assistant frontend will be accessible locally from your browser at [http://localhost:8123](http://localhost:8123)
```

Comment:
AFAIK the single quote char ' is generally used to identify minutes, while the double quote " seconds. Actually now that I'm digging into it, [according to wikipedia](https://en.wikipedia.org/wiki/Prime_(symbol)) it should be a "prime" char instead.
Anyway, I'll put _minutes_ just to avoid any confusion.